core/core-api/src/main/java/com/hbc/pms/core/api/config/WebSocketConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.hbc.pms.core.api.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker;
+import org.springframework.web.socket.config.annotation.StompEndpointRegistry;
+import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer;
+
+@Configuration
+@EnableWebSocketMessageBroker
+public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {
```

Comment:
Please document your findings in confluence. Things such as data the data flow, diagram etc... should be added to the document

Reply:
1. Origin: I've configured the WebSocketConfig to allow the localhost origin only. I think these allowed origins should be put in a constant file, but I still don't know the appropriate path. I will update as soon as I can when find out.
2. Lombok: I've replaced the logger with the @Slf4j Lombok's annotation. And I see some places in the sample code still not using Lombok yet (e.g., PlcService.java, ApiControllerAdvice.java, etc.). Would you like me to update this also? 
3. Document: I mostly follow the two tuts mentioned here (https://hbc-capstone.atlassian.net/l/cp/krMW2bHc). But I also think documenting and diagraming the flow is a good idea for me to understand better. Regarding this, I will try to complete it before **midnight**

Reply:
https://hbc-capstone.atlassian.net/l/cp/028A1sT5
Hi @huybui38, I've captured the flow investigated in the code runtime. I really appreciate any of your comments, no matter how harsh they are, so I can understand more and produce better code. 
If everything looks fine with the sample code, I will discuss further with you and Kien the approach of handling multiple stations' data. 
Good night!